### PR TITLE
Refactoring: PHP 7.1, Bootstrap3/4 Renderers

### PR DESCRIPTION
**Pull request by mabar**
@f3l1x Code is ready for review.

Note to self: Documentation is needed.

### CODE REVIEW

**Comment by f3l1x** [comment]
Thank you @mabar. Could you also update ruleset.xml to force tabs (https://github.com/contributte/console/blob/master/ruleset.xml#L16-L22). Thanks.

**Comment by f3l1x** [comment]
CI is failing: https://travis-ci.org/contributte/forms/jobs/367030945

**Comment by aleswita** [comment]
Hi @mabar 
what you think about addons option for Bootstrap 4 from this repository: https://github.com/aleswita/FormRenderer/blob/master/src/BootstrapV4Renderer.php#L149

Example:
vystrizek

**Comment by f3l1x** [comment]
Could you guys collaborate together? @aleswita + @mabar?

**Comment by f3l1x** [comment]
@mabar Please use latest ninjify/qa (you know) and finish this PR to be merged ;-) Thanks.

File: ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<ruleset name="Contributte">
+    <!-- Extending rulesets -->
+    <rule ref="vendor/ninjify/coding-standard/ruleset.xml"/>
```
Comment (f3l1x) [inline ruleset.xml]: Use `contributte.xml` instead of ruleset.xml ;-) It force tabs.

File: src/Renderers/Bootstrap3HorizontalRenderer.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Forms\Renderers;
+
+use Nette\Forms\Controls;
+use Nette\Forms\Form;
+use Nette\Forms\Rendering\DefaultFormRenderer;
+use function mb_strpos;
```
Comment (f3l1x) [inline src/Renderers/Bootstrap3HorizontalRenderer.php]: Please, drop use functions. We've discussed it on Slack. You know.

**Comment by f3l1x** [comment]
Awesome, thanks.

**Pull request merged**